app/models/invenio_rdm_record_converter.rb

```diff
@@ -15,6 +14,7 @@ class InvenioRdmRecordConverter < Sufia::Export::Converter
   # @param [GenericFile] generic_file file to be converted for export
   def initialize(generic_file=nil)
     return unless generic_file
+    @@header_lookup = HeaderLookup.new
```

Comment:
Allocate the HeaderLookup as a class variable so it only needs to be allocated once. This should actually be changed to `||=` so I'll do that quick.